libs/application/ui-shell/src/components/ScreenFooter.tsx

```diff
@@ -110,6 +114,21 @@ export const ScreenFooter: FC<FooterProps> = ({
           <Box display="inlineFlex" padding={2} paddingRight="none">
             {hasSubmitField ? (
               renderSubmitButtons()
+            ) : isLastScreen ? (
```

Comment:
Just one question regarding the `renderLastScreenButton`, was the [`buildSubmitField`](https://github.com/island-is/island.is/blob/main/libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts#L719-L731) not answering your problem?

Reply:
Unfortunately no. This is the screen **after** the submit form screen.